src/connectors/refinement-list/connectRefinementList.ts

```diff
@@ -301,20 +304,17 @@ const connectRefinementList: RefinementListConnector = function connectRefinemen
                 }))
               );
 
-              const canToggleShowMore =
-                isShowingMore && lastItemsFromMainSearch.length > limit;
-
               renderFn(
                 {
                   ...widget.getWidgetRenderState({
                     ...renderOptions,
                     results: lastResultsFromMainSearch,
                   }),
                   items: normalizedFacetValues,
-                  canToggleShowMore,
+                  canToggleShowMore: false,
```

Comment:
How do you know it's `false`?

Reply:
show more doesn't display when in search for facet values, it can't work (would need to do a new search). If you call the method today the query will disappear

Reply:
Ah it makes sense.